package.json

```diff
@@ -23,6 +23,7 @@
     "dotenv-cli": "^6.0.0",
     "formik": "^2.2.9",
     "html-react-parser": "^3.0.4",
+    "i18next": "^22.4.10",
```

Comment:
Do you need them both? `i18next` and `react-i18next`

Reply:
I used both. According to their documentation i18n-next is the core library that sets up the language support. The extension for React adds a hook method so the translate function (t) can easily be called on in components to use the translate functionality.

Reply:
Probably could do without, but should be more optimized.

Reply:
Ok clear